network/network_test.go

```diff
@@ -50,6 +50,12 @@ func TestGatewayLoading(t *testing.T) {
 	}
 }
 
+func TestVerifyConnectivity(t *testing.T) {
```

Comment:
Why are we change the network test coverage in this pull request?

Reply:
I'll drop this.